src/main/java/nextstep/subway/path/ui/PathController.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.subway.path.ui;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import nextstep.subway.path.application.PathService;
+import nextstep.subway.path.dto.PathRequest;
+import nextstep.subway.path.dto.PathResponse;
+
+@RequestMapping("/paths")
+@RestController
+public class PathController {
+
+	private final PathService pathService;
+
+	public PathController(final PathService pathService) {
+		this.pathService = pathService;
+	}
+
+	@GetMapping
+	public ResponseEntity<PathResponse> getShortestPath(
```

Comment:
```suggestion
	public ResponseEntity<PathResponse> getShortestPath(PathRequest pathRequest) {
	...
	}
```

이렇게 작성해도 될 것 같은데, 특정 컨벤션을 지키고 계신걸까요?

그리고 커밋 전에 인텔리제이(mac)를 혹시 사용중이시라면 `option` + `command` + `L` 로 코드 정리를 할 수 있습니다.